src/main/java/com/google/cloud/tools/appengine/api/deploy/DefaultGenRepoInfoFileConfiguration.java

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
This command currently exists in app, which is why I thought this was the correct package.

However, I hear the debug command is the real deal, so I can also move this to its own debug package.